package_manifest.yaml

```diff
@@ -39,6 +42,12 @@ foreman_scl_packages:
         - el7-extras
         - el7-puppet-6
         - el7-scl
+      rhel8:
```

Comment:
So, based on the error I see in the jenkins job, it appears it's looking for a key of `el8`, not `rhel8`, perhaps we can try that to see if it works?  For here and for above